task/buildah-oci-ta/0.1/buildah-oci-ta.yaml

```diff
@@ -125,6 +130,10 @@ spec:
       secret:
         optional: true
         secretName: $(params.ENTITLEMENT_SECRET)
+    - name: optional-secret
+      secret:
+        optional: true
+        secretName: $(params.OPTIONAL_SECRET)
```

Comment:
I suspect the test failures may be caused by 

```yaml
    - name: optional-secret
      secret:
        optional: true
        secretName: ""  # the default $(params.OPTIONAL_SECRET) is ""
```

Reply:
Attempted providing a default value in ef5f5de.